app/services/host_inventory_api.rb

```diff
@@ -24,6 +24,14 @@ def host_already_in_inventory(hostname_or_id)
     find_results(JSON.parse(response.body))
   end
 
+  def inventory_host
+    @inventory_host ||= host_already_in_inventory(@id) ||
+                        (@hostname && host_already_in_inventory(@hostname))
+  end
+
+  # This function is meant to be only used for testing purposes within a
+  # development environment. It is not and should not be called from anywhere
+  # else than a local rake task or the Rails console itself.
   def create_host_in_inventory
```

Comment:
I use httpie for this. I want to push for just removing it, but I can't really make a good case for why leaving this here is bad, other than it's no longer used. :man_shrugging: 

```
http --verify no --auth <auth> POST https://ci.cloud.redhat.com/api/inventory/v1/hosts < systems.json
HTTP/1.1 207 MULTI STATUS
Transfer-Encoding: chunked
X-Powered-By: Express
connection: close
content-type: application/json
date: Wed, 11 Mar 2020 16:15:39 GMT
server: openresty/1.13.6.1
set-cookie: bd2e829019cc5851647126f4f0834941=546c250d4ef07df905cfb33195ad89ad; path=/; HttpOnly; Secure
x-content-type-options: nosniff
x-fakamai-app-hint: API
x-fakamai-downstream-host: https://api-gateway-ci.5a9f.insights-dev.openshiftapps.com
x-fakamai-env: ci
x-rh-insights-request-id: 4400338bbd814d0f88a73882ba2894f8

{
    "data": [
        {
            "host": {
                ...
            },
            "status": 200
        }
    ],
    "errors": 0,
    "total": 1
}
```

With systems.json like this:

```js
[
  {
    "account": "1212729",
    "display_name": "rhel7-insights-client-fake100.virbr0.akofink-laptop",
    "facts": [],
    "fqdn": "rhel7-insights-client-fake001.virbr0.akofink-laptop",
    "reporter": "akofink's local dev machine",
    "stale_timestamp": "2021-03-04T16:15:44Z"
  }
]
```